GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1744,20 +1744,7 @@ private void Unstage(bool canUseUnstageAll = true)
                             continue;
                         }
 
-                        var item2 = item;
-                        if (unstagedFiles.Exists(i => i.Name == item2.Name))
-                        {
-                            continue;
-                        }
```

Comment:
Just curious: What was the purpose of the cloned identifier `item2`, which was not used somewhere else?

Reply:
Should probably have been removed in a refactoring...